dead_man_button: 4
next_robot_button: 5
gear_up_button: 6
gear_down_button: 7
board_button: 10
ad_button: 11
steering: [3]
steering_aux: 2
accel: [1]
